data/json/furniture_and_terrain/terrain-walls.json

```diff
@@ -16,27 +16,103 @@
   },
   {
     "type": "terrain",
-    "id": "t_wall",
-    "name": "wall",
-    "description": "A standard wall consisting of a wooden support structure filled with insulation and drywalled.  Painted a neutral off-white or cream color, it could use some more vibrant paint.  Looks flammable.",
-    "symbol": "#",
+    "abstract": "t_abstract_column",
+    "name": "abstract column",
+    "description": "Encompasses rock columns.  If you see this in-game, it's a bug.",
```

Comment:
```suggestion
    "description": { "str": "Encompasses rock columns.  If you see this in-game, it's a bug.", "//~": "NO_I18N" },
```

Reply:
Please also apply this `NO_I18N` to all other abstracts.

Reply:
I tried doing this with just `description`, it didn't work, and then with just `name` (like with the `ammolink` abstract) and that didn't work either. Do I need both?

Reply:
nvm, the translation parser was the issue